gnirehtet.cmd

```diff
@@ -0,0 +1,150 @@
+@ECHO OFF
+CALL :locate_file ADB %~dp0adb.exe "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
```

Comment:
`:locate_file ` doesn't searching in `%PATH%`. In windows, unlike Linux, adb and Java doesn't automatically added to the Path environment variable. They only way to add these is manually, which is isn't so easy. In this script I'm using the default location of the binaries of adb and Java which I believe is more likely to be.  If I see people asking for it, I'll rewrite `:locate_file `, for now I see no reason.